### DISCUSSION TITLE
SEB with windows taskbar?

### DISCUSSION THREAD

**Discussion by kstarzecki** [SafeExamBrowser/seb-win-refactoring#918] [General]
Hey, would it be possible to run SEB with windows taskbar?

We're thinking about managing the machines via windows policies. This way we could restrict the app/directory access. We would like to use SEB as a "default browser", while maintaining the windows taskbar (for work folder access, permitted app access, etc).

In short: windows policies to restrict machine access, and SEB to restrict browser access. 

Would that be possible in some way?

**Comment by dbuechel** [comment]
That should most definitely be possible, you can disable the SEB taskbar under User Interface > Show taskbar and set the kiosk mode to None under Security (see https://safeexambrowser.org/windows/win_usermanual_en.html#AppearancePane and https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane respectively).

**Reply by kstarzecki** [reply]
Nice, thanks! Will post results once we have it in a working state, for anyone interested. (probably will take a while).

**Reply by dbuechel** [reply]
Great, yes that would certainly be helpful for other users.
